openedx/adg/lms/applications/models.py

```diff
@@ -50,6 +50,20 @@ class UserApplication(TimeStampedModel):
     )
     cover_letter = models.TextField(blank=True, )
 
+    OPEN = 'open'
+    ACCEPTED = 'accepted'
+    WAITLIST = 'waitlist'
+
+    STATUS_CHOICES = (
+        (OPEN, _('Open')),
+        (ACCEPTED, _('Accepted')),
+        (WAITLIST, _('Waitlist')),
+    )
```

Comment:
Can we please create a new file `constants.py` in the same directory as this file and move these three variables and the `STATUS_CHOICES` tuple to that file. We can import them in `models.py` and use as needed. If `STATUS_CHOICES` is moved to `constants.py`, to give better context, it should be renamed to `APPLICATION_STATUS_CHOICES`.

Reply:
From django docs, defining the choices and names for each choice inside the model class keeps all of that information with the class that uses it, and helps reference the choices. I think it's more neat than keeping all constants together.